tests/v2/validation/rbac/rbac.go

```diff
@@ -257,3 +259,78 @@ func createRole(client *rancher.Client, context string, roleName string, rules [
 	return
 
 }
+
+func waitForActiveCluster(client *rancher.Client, clusterID string) error {
+	err := kwait.Poll(500*time.Millisecond, 30*time.Minute, func() (done bool, err error) {
+		client, err = client.ReLogin()
+		if err != nil {
+			return false, err
+		}
+		clusterResp, err := client.Management.Cluster.ByID(clusterID)
+		if err != nil {
+			return false, err
+		}
+		if clusterResp.State == "active" {
+			return true, nil
+		}
+		return false, nil
+	})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func editCluster(client *rancher.Client, clustername string, namespace string, psact string) (clusterType string, err error) {
+	clusterID, err := clusters.GetClusterIDByName(client, clustername)
+	if err != nil {
+		return "", err
+	}
+	//Check if the downstream cluster is RKE2/K3S or RKE1
+	if strings.Contains(clusterID, "c-m-") {
+		clusterType = "RKE2K3S"
+		clusterObj, existingSteveAPIObj, err := clusters.GetProvisioningClusterByName(client, clustername, namespace)
+		if err != nil {
+			return "", err
+		}
+		oldPsact := clusterObj.Spec.DefaultPodSecurityAdmissionConfigurationTemplateName
+		clusterObj.Spec.DefaultPodSecurityAdmissionConfigurationTemplateName = psact
+		_, err = clusters.UpdateK3SRKE2Cluster(client, existingSteveAPIObj, clusterObj)
+		if err != nil {
+			return clusterType, err
+		}
+		//Revert the value of psact back to the original value
+		clusterObj.Spec.DefaultPodSecurityAdmissionConfigurationTemplateName = oldPsact
+		_, err = clusters.UpdateK3SRKE2Cluster(client, existingSteveAPIObj, clusterObj)
+		if err != nil {
+			return clusterType, err
+		}
+	} else {
+		clusterType = "RKE"
+		if psact == "" {
+			psact = " "
```

Comment:
I guess this if loop is not needed since we are passing in the value of psact to be "" in the function call, we can change it to psact = " "..
Also, just curious, doesn't psact= "" work for RKE1 clusters?